contents/docs/error-tracking/upload-source-maps/angular.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Upload source maps for Angular
+---
+
+import { CallToAction } from 'components/CallToAction'
+import { Steps, Step } from 'components/Docs/Steps'
+import CLIDownload from '../_snippets/cli/download.mdx'
+import CLIAuthenticate from '../_snippets/cli/authenticate.mdx'
+import InjectSourceMap from './_snippets/inject-source-map.mdx'
+import UploadSourceMap from './_snippets/upload-source-map.mdx'
+
+> By the end of this guide, you'll understand how to upload source maps for your framework and have proper stack traces in PostHog.
+>
+> <CallToAction type="primary" to="https://us.posthog.com/onboarding/error_tracking?step=source_maps">
+>    Follow in PostHog
+> </CallToAction>
```

Comment:
I get what we're trying to do with this callout but it feels like we're adding a layer of indirection when the content is already in front of the user. I would remove these and focus on a way of merging the docs / in-app content (future work)